timeline-chart/src/time-graph-unit-controller.ts

```diff
@@ -45,6 +63,24 @@ export class TimeGraphUnitController {
         }
     }
 
+    /**
+     * Handles changes on world range change, but before the states/rows render.
+     * If the behavior looks weird  or glitchy after implementation and you're working in 
+     * a TimeGraphViewportLayer / TimeGraphChartLayer class, consider using:
+     * 
+     * TimeGraphChartLayer#onWorldRangeChanged
+     */
+    onWorldRangeChanged(handler: (worldRange: TimelineChart.TimeGraphRange) => void) {
```

Comment:
I thought this would be needed, but everything worked best with TimeGraphViewportContainer#handleOnWorldRender vs onWorldRangeChange.  I can remove the handlers, but there might be a use for it in the future. UnitController.onZoomChange was not used before this, but there was code there to handle it (sort of).

Reply:
Apparently not in use. I agree with Bernd, it should be removed even if may be of use in the future.